core/common.js

```diff
@@ -44,3 +46,58 @@ const setMainWorkspace = function(workspace) {
   mainWorkspace = workspace;
 };
 exports.setMainWorkspace = setMainWorkspace;
+
+/**
+ * Container element in which to render the WidgetDiv, DropDownDiv and Tooltip.
+ * @type {?Element}
+ */
+let parentContainer;
+
+/**
+ * Get the container element in which to render the WidgetDiv, DropDownDiv and\
+ * Tooltip.
+ * @return {?Element} The parent container.
+ */
+const getParentContainer = function() {
+  return parentContainer;
+};
+exports.getParentContainer = getParentContainer;
+
+/**
+ * Set the parent container.  This is the container element that the WidgetDiv,
+ * DropDownDiv, and Tooltip are rendered into the first time `Blockly.inject`
+ * is called.
+ * This method is a NOP if called after the first ``Blockly.inject``.
+ * @param {!Element} newParent The container element.
+ */
+const setParentContainer = function(newParent) {
+  parentContainer = newParent;
+};
+exports.setParentContainer = setParentContainer;
+
+/**
+ * All of the connections on blocks that are currently being dragged.
+ * @type {!Array.<!Connection>}
```

Comment:
We use {!Array<!Connection>} now, rather than {!Array.<!Connection>}
There was a mass change a couple of months ago as the dots were eradicated across the codebase.
Here and twice below.